src/lib/output/renderer.ts

```diff
@@ -201,7 +201,24 @@ export class Renderer extends ChildableComponent<Application, RendererComponent>
                 }
             }
 
-            this.theme = this.addComponent('theme', new DefaultTheme(this, path));
+            let filename = Path.join(path, 'theme.js');
+            if (!FS.existsSync(filename)) {
+                this.theme = this.addComponent('theme', new DefaultTheme(this, path));
+            } else {
+                try {
+                    const plugin = require(filename);
+                    if(typeof plugin === 'function') {
+                        plugin(this.application);
```

Comment:
Just the same code as in https://github.com/TypeStrong/typedoc/pull/504#discussion_r116868923? Check `exports.default` or use `exports` directly based on existence of a function. I'm not sure that using the direct result of `require` is a good idea, it'd make it non-trivial for anyone writing in TypeScript to actually write a plugin, which is why I suggest keeping the `.default` handling around. I can just change the code myself in the future if that's easier.